### PR TITLE
README: Add Communication section with Forum information

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README. Similar PRs are being raised across all included collections under the ansible-collection org for now.

If you have other tags/forum groups related to the collection I didn't spot, please update corresponding lines by suggesting changes to the PR.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

**Review by mariolenz — APPROVED** [review]
@bardielle @machacekondra @mikemorency LGTM, what do you think?

**Review by mikemorency — APPROVED** [review]
LGTM, we are going to update the README this release anyway so Ill make sure to include the working group section then

File: README.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ This repo hosts the `vmware.vmware` Ansible Collection.
 
 The collection includes the VMware modules and plugins to help the management of VMware infrastructure.
 
+## Communication
+
+* Join the Ansible forum:
+  * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
+  * [Posts tagged with 'vmware'](https://forum.ansible.com/tag/vmware): subscribe to participate in collection-related conversations.
```
Comment (mariolenz) [inline README.md:11]:
```suggestion
  * [Posts tagged with 'vmware'](https://forum.ansible.com/tag/vmware): subscribe to participate in collection-related conversations.
  * [Ansible VMware Automation Working Group](https://forum.ansible.com/g/ansible-vmware): by joining the team you will automatically get subscribed to the posts tagged with ['vmware'](https://forum.ansible.com/tag/vmware).
```

@Andersson007 Would this be the correct way to mention the forum group?

**Comment by mariolenz** [comment]
> Please don't merge yet. I'd like to get an answer on my last comment first.

@bardielle @machacekondra @mikemorency My comment / suggested change has been committed by @Andersson007. Should we merge? LGTM now.

**Comment by Andersson007** [comment]
Thanks everyone!

**Pull request merged**